ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py

```diff
@@ -55,8 +55,9 @@ def get_changes_in_pr(data):
     master_branch = subprocess.run(command, text=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     devel_branches = ""
     for branch in master_branch.stdout.split("\n"):
-        if "devel" in branch:
+        if "devel" in branch or "master" in branch or "main" in branch:
```

Comment:
Commented on slack but anyways would be better to if branch in ["devel", "master", "main"]

Reply:
agreed